### PR TITLE
Typings for src/processing/script (-70 semantic errors)

### PR DESCRIPTION
**Pull request by LavrovArtem**

### CODE REVIEW

**Comment by Farfurix** [comment]
@testcafe-build-bot retest

**Comment by Farfurix** [comment]
@testcafe-build-bot retest

File: src/processing/script/instrumented.ts [hunk 1]
```diff
@@ -32,10 +32,10 @@ const test = (regexp: RegExp, str: string) => regexp.test ? regexp.test(str) : r
 // cases like `WRAPPABLE_METHOD['toString']` will fail.
 // We could use the hasOwnProperty test, but it is
 // significantly slower than the regular expression test
-export function shouldInstrumentMethod (name: string|number|boolean|RegExp): boolean {
+export function shouldInstrumentMethod (name: string|number|boolean|RegExp|null|void): boolean {
```
Comment (miherlosev) [inline src/processing/script/instrumented.ts]: I guess we can simplify to `name: any`.

**Comment by LavrovArtem** [comment]
@testcafe-build-bot retest

**Pull request merged**